File: app/src/main/java/com/example/seyahatkolaymobileapp/MainActivity.java
package com.example.seyahatkolaymobileapp;

import android.content.Intent;
import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;
import com.example.seyahatkolaymobileapp.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    ActivityMainBinding binding;

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        replaceFragment(new TicketFragment());

        binding.bottomNavigationView.setOnItemSelectedListener(menuItem -> {
            if (menuItem.getItemId() == R.id.fragment_ticket) {
                replaceFragment(new TicketFragment());
            } else if (menuItem.getItemId() == R.id.fragment_aboutus) {
                replaceFragment(new AboutusFragment());
            }
            return true;
        });
    }
    public void replaceFragment(Fragment fragment) {
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        fragmentTransaction.replace(R.id.frame_layout, fragment);
        fragmentTransaction.commit();
    }
}

File: app/src/main/java/com/example/seyahatkolaymobileapp/MainActivity.kt
package com.example.seyahatkolaymobileapp

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.example.seyahatkolaymobileapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    var binding: ActivityMainBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(
            layoutInflater
        )
        setContentView(binding!!.root)
        replaceFragment(TicketFragment())

        binding!!.bottomNavigationView.setOnItemSelectedListener { menuItem: MenuItem ->
            if (menuItem.itemId == R.id.fragment_ticket) {
                replaceFragment(TicketFragment())
            } else if (menuItem.itemId == R.id.fragment_user_profile) {
                replaceFragment(UserProfileFragment())
            } else if (menuItem.itemId == R.id.fragment_aboutus) {
                replaceFragment(`AboutusFragment`())
            }
            true
        }
    }

    fun replaceFragment(fragment: Fragment?) {
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.frame_layout, fragment!!)
        fragmentTransaction.commit()
    }
}
